### PR TITLE
test: add folders and move test files

### PR DESCRIPTION
**Pull request by kenjis**
Too many files in a folder.

### CODE REVIEW

File: tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ protected function setUp(): void
 
         $config = new Auth();
         $auth   = new Authentication($config);
-        $auth->setProvider(model(UserModel::class)); // @phpstan-ignore-line
+        $auth->setProvider(\model(UserModel::class)); // @phpstan-ignore-line
```
Comment (MGatner) [inline tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php:31]: Our style guide decided against slashing root functions. Did you add these, or did they slip in via CS Fixer/Rector somehow?

**Review by MGatner — CHANGES_REQUESTED** [review]

**Review by MGatner — CHANGES_REQUESTED** [review]
I'm good with the folder move but please revert the function slashing. If you have a compelling case for them we should discuss it for the style guide first.

File: tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php (hunk 1 above)
Comment (kenjis) [reply tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php:31]: It seems PhpStorm did it when refactoring.

Comment (kenjis) [reply tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php]: Removed.

**Review by MGatner — APPROVED** [review]

**Pull request merged**